src/goto-symex/symex_builtin_functions.cpp

```diff
@@ -434,10 +434,8 @@ void goto_symext::symex_cpp_new(
   exprt rhs(ID_address_of, pointer_type);
 
   if(do_array)
-  {
-    index_exprt index_expr(symbol.symbol_expr(), from_integer(0, index_type()));
-    rhs.move_to_operands(index_expr);
-  }
+    rhs.add_to_operands(
```

Comment:
AFAIK usually keep braces when the if statement is multi-line, even if it's a single statement

Reply:
Fixed!